reasoner/runner/runner-common/src/test/java/com/antgroup/openspg/reasoner/graphstate/KgstateGraphStateTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 Ant Group CO., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied.
+ */
+package com.antgroup.openspg.reasoner.graphstate;
+
+import com.antgroup.openspg.reasoner.common.graph.property.IProperty;
+import com.antgroup.openspg.reasoner.common.graph.vertex.IVertex;
+import com.antgroup.openspg.reasoner.common.graph.vertex.IVertexId;
+import com.antgroup.openspg.reasoner.graphstate.impl.KgStateSourceGraphState;
+import com.antgroup.openspg.reasoner.warehouse.common.AbstractGraphLoader;
+import java.util.HashSet;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class KgstateGraphStateTest {
+  @Test
+  public void testReadVertex() {
+    KgStateSourceGraphState kgStateSourceGraphState = new KgStateSourceGraphState();
```

Comment:
change class name KgStateSourceGraphState to GraphStoreGraphState

Reply:
done